src/main/resources/gwwhit.mixins.json

```diff
@@ -1,4 +1,42 @@
 {
+	"required": true,
+	"minVersion": "0.8",
+	"package": "fr.anatom3000.gwwhit.mixin",
+	"compatibilityLevel": "JAVA_8",
+	"mixins": [
+		"AnimalEntityMixin",
+		"ChatMessageC2SPacketMixin",
+		"ChunkGeneratorMixin",
+		"ChunkGeneratorSettingsMixin",
+		"DimensionTypeMixin",
+		"FixedYOffsetMixin",
+		"ItemMixin",
+		"LivingEntityAccessor",
+		"LivingEntityMixin",
+		"LootManagerMixin",
+		"NoiseChunkGeneratorMixin",
+		"PlayerManagerMixin"
+	],
+	"client": [
+		"ChatHudMixin",
+		"ChatScreenMixin",
+		"ClientWorldMixin",
+		"Deadmau5FeatureRendererMixin",
+		"GameRendererMixin",
+		"KeyboardMixin",
+		"LanguageMixin",
+		"LivingEntityRendererMixin",
+		"MatrixStackMixin",
+		"MixinFrustum",
+		"PlayerEntityModelMixin",
+		"ReloadableResourceManagerImplMixin",
+		"SoundSystemMixin",
+		"TitleScreenMixin"
+	],
+	"injectors": {
+		"defaultRequire": 1
+	}
+.,=====
```

Comment:
this line shouldn't look like that

Reply:
wops, didn't remove it from the merge conflict resolve